main/static/home.css

```diff
@@ -26,51 +26,95 @@
 	margin-right: 5px;
 }
 a.home-button:hover {
-	background-color: #333;
-	color: #eee;
+	color: black !important;
+	border-color: black;
+	box-shadow: 0px 0px 5px 0px #444;
 
 	text-decoration: none;
 }
 
-section#section-whatis-enactus {
-	/*background: url("/static/what_is_enactus.png") no-repeat;
-	background-size: cover;
-	background-attachment: fixed;*/
+#section-coverpics {
+	margin: 0px;
+	padding: 0px;
+}
 
-	text-align: left;
+#section-coverpics #carousel-main {
+	width: 100%;
+}
+#carousel-main .item {
+	width: 100%;
+	height: 650px;
+	text-align: center;
+	overflow: hidden;
+}
 
-	background-color: #FFF7E0;
+#carousel-main .carousel-control {
+	color: #222;
+	background: transparent;
 }
-#section-whatis-enactus .section-content {
-	margin: 10px 0px;
+
+#carousel-main .carousel-caption .carousel-caption-title {
+	font-size: 26pt;
+	color: #222;
 }
 
-#section-whatis-enactus .section-text {
-	margin-top: 10px;
+#carousel-main .carousel-title-rect {
+	background-color: #FEC222;
+	display: inline-block;
+	padding: 5px 10px;
 }
 
-@media (max-width: 768px) {
-	section#section-whatis-enactus h1 {
-		font-size: 42pt;
-	}
-	#section-whatis-enactus h1 #firsthalf{
-		font-size: 32pt;
+#carousel-main .carousel-subtitle-rect {
+	background-color: #333;
+	color: #eee;
+	font-size: 16pt;
+	display: inline-block;
+	padding: 5px 10px;
+}
+.carousel-title-rect,.carousel-subtitle-rect {
+	box-shadow: 0px 0px 5px #888;
+	text-shadow: none;
+}
+
+#carousel-main .carousel-caption {
+	text-align: left;
+}
+
+@media (max-width: 768px) { /* phones */
+	#carousel-main .item {
+		height: 250px;
 	}
 }
 
-#section-whatis-enactus h1 {
-	background-color: #FEC222;
+#carousel-main .item img {
+	width: 100%;
+	height: auto;
 
-	display: inline;
+	position: relative;
+  top: 50%;
+  -webkit-transform: translateY(-50%);
+  -ms-transform: translateY(-50%);
+  transform: translateY(-50%);
```

Comment:
It's indented with two spaces, and tabs appear on my editor as two spaces so it lines up. We should really stick to a standard. For now, I'll replace this with a tab since that's the norm in this particular file.